repoauth/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name='pulp-repoauth',
+    version='2.6.0',
+    license='GPLv2+',
+    packages=find_packages(exclude=['test']),
+    author='Pulp Team',
+    author_email='pulp-list@redhat.com',
+    entry_points={
+        'console_scripts': [
```

Comment:
does this really belong here? I would expect it to only come with the server package.